hack/lib/golang.sh

```diff
@@ -433,6 +433,44 @@ kube::golang::get_physmem() {
   echo 1
 }
 
+
+kube::golang::verify_code_builds() {
+  if ! OUT=$(which gb 2>&1); then
+    hack/build-go.sh
```

Comment:
I see you did that later, not sure which is better - your call. I really only want to see warnings if I am doing something sub-optimal...